screen/PreviousResult/PrveiousResultScreen.js

```diff
@@ -1,13 +1,107 @@
-import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  Button,
+  ActivityIndicator,
+  StyleSheet,
+} from 'react-native';
 
-import COLORS from '../../common/constants/COLORS';
+import StandardModal from '../../common/components/StandardModal';
+import TitleText from '../../common/components/TilteText';
+import UnitText from '../../common/components/UnitText';
+import ErrorMessage from '../../common/components/ErrorMessage';
 import FONT from '../../common/constants/FONT';
+import COLORS from '../../common/constants/COLORS';
+
+const SURVIVED = "YOU'RE SURVIVED";
+const FAILED = 'YOU WERE INFECTED';
+const KILOMETER = 'km';
+const TIME = 'm';
+
+const PreviousResultScreen = () => {
+  const [modalVisible, setModalVisible] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  let record = {
+    distance: 0,
+    time: 0,
+    speed: 0,
+    solo: {
+      isWinner: false,
+    },
+  };
+
+  const openModalHandler = async () => {
+    setModalVisible(true);
+    try {
+      const response = await fetch('backendAddress', {
+        method: 'GET',
+      });
+
+      setIsLoading(true);
+      record = response.json();
+      setIsLoading(false);
+      setIsError(true);
+
+      return record;
+    } catch (error) {
+      setIsLoading(false);
+      return ErrorMessage(error.message);
+    }
+  };
```

Comment:
```suggestion
  const openModalHandler = async () => {
    setIsError(false);
    setIsLoading(true);
    try {
      const response = await fetch('backendAddress', {
        method: 'GET',
      });


      record = await response.json();
      setIsLoading(false);
      setModalVisible(true);

      return record;
    } catch (error) {
      setIsError(true);
      setIsLoading(false);
      return ErrorMessage(error.message);
    }
  };
```
모달의 처음 상태값이 트루여서 아마 이 로직과 맞물려 에러가 발생해도 Modal이 켜질거 같습니다. 
위에 피드백 드린 부분과 지금 고쳐준 로직을 바탕으로 다시 작성하시면 올바르게 작동할거 같습니다.
useState의 초기 값 설정에 주의 하시길 바랍니다.

Reply:
네 다시 보니까 로직이 이상했네요 그 부분과 아래부분까지 수정하겠습니다 감사합니다